include/sdk/gx/GXLight.h

```diff
@@ -0,0 +1,18 @@
+#ifndef GX_LIGHT_H
+#define GX_LIGHT_H
+
+#include <sdk/GXTypes.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void GXSetChanMatColor(int, _GXColor color);
+void GXSetNumChans(int nChans);
```

Comment:
void GXSetNumChans(u8 num);

Reply:
The name of the parameter is `nChans`, as evidenced by the following `OSPanic` call:

```cpp
OSPanic("GXLight.c", 872, "GXSetNumChans: nChans > 2");
```